network-api/networkapi/wagtailpages/pagemodels/blog/blog.py

```diff
@@ -358,3 +367,9 @@ def get_meta_description(self):
                 return truncatechars(text, 153)
 
         return super().get_meta_description()
+
+
+# Overriding wagtail-metadata's search_image and search_description
+# fields in the promote panel to make them required for blog pages.
+BlogPage._meta.get_field("search_image").blank = False
+BlogPage._meta.get_field("search_description").blank = False
```

Comment:
Since there were some Blog page specific requests, these live under blog.py

Reply:
This is outdated as it makes these fields required for all pages instead of just the blog page.
We are now making the desired changes in the `BlogPages` custom [BlogPageForm](https://github.com/mozilla/foundation.mozilla.org/blob/da8b6fb62a2bddab24913644d481b502f5d702e3/network-api/networkapi/wagtailpages/forms.py#L29)